policy-controller/k8s/status/src/index.rs

```diff
@@ -158,7 +164,7 @@ impl Index {
 
     // If the route is new or its parents have changed, return true so that a
     // patch is generated; otherwise return false.
-    fn update_http_route(&mut self, id: ResourceId, parents: Vec<ParentReference>) -> bool {
+    fn update_http_route(&mut self, id: ResourceId, parents: References) -> bool {
```

Comment:
the `References` argument should probably not be called `parents` anymore, as it now includes `BackendReference`s.

Reply:
oops